rfc6962-bis.xml

```diff
@@ -541,9 +541,7 @@ ranges.
         v2(1), (255)
     } Version;
 
-    struct {
-        opaque key_id[HASH_SIZE];
-    } LogID;
+    opaque LogID&lt;1..2^8-1&gt;;
```

Comment:
Oh, I see - it's an opaque field of 1 - 255 octets.
Not sure how that fits with the DER encoding? i.e. should it / could it be converted to an ASN.1 OID and if so, how?